libember_slim/Source/ber.h

```diff
@@ -20,6 +20,12 @@
 #ifndef __LIBEMBER_SLIM_BER_H
 #define __LIBEMBER_SLIM_BER_H
 
+#ifdef LIBEMBER_DLL_EXPORTS
+#define LIBRARY_API __declspec(dllexport)

```

Comment:
Could you please replace these inline definitions with a more complete implementation analogous to the variant found in [the C++ Ember+ library](../../blob/master/libember/Headers/ember/util/Api.hpp) and implemented a separate, internal header file.